# SOME DESCRIPTIVE TITLE.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PostgreSQL 9.0 beta 3\n"
"Report-Msgid-Bugs-To: pgsql-bugs@postgresql.org\n"
"POT-Creation-Date: 2010-07-20 12:54+0900\n"
"PO-Revision-Date: 2010-07-15 19:00+0900\n"
"Last-Translator: HOTTA Michihide <hotta@net-newbie.com>\n"
"Language-Team: jpug-doc <jpug-doc@ml.postgresql.jp>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: fe-auth.c:242
#, c-format
msgid "could not set socket to blocking mode: %s\n"
msgstr "ソケットをブロッキングモードに設定できませんでした: %s\n"

#: fe-auth.c:260 fe-auth.c:264
#, c-format
msgid "Kerberos 5 authentication rejected: %*s\n"
msgstr "Kerberos 5認証が拒絶されました: %*s\n"

#: fe-auth.c:290
#, c-format
msgid "could not restore non-blocking mode on socket: %s\n"
msgstr "ソケットを非ブロッキングモードに戻すことができませんでした: %s\n"

#: fe-auth.c:403
msgid "GSSAPI continuation error"
msgstr "GSSAI続行エラー"

#: fe-auth.c:431 fe-auth.c:657
msgid "host name must be specified\n"
msgstr "ホスト名を指定しなければなりません\n"

#: fe-auth.c:438
msgid "duplicate GSS authentication request\n"
msgstr "重複するGSS認証要求\n"

#: fe-auth.c:458
msgid "GSSAPI name import error"
msgstr "GSSAPI名のインポートエラー"

#: fe-auth.c:544
msgid "SSPI continuation error"
msgstr "SSPI続行エラー"

#: fe-auth.c:555 fe-auth.c:629 fe-auth.c:663 fe-auth.c:760 fe-connect.c:1802
#: fe-connect.c:3129 fe-connect.c:3346 fe-connect.c:3762 fe-connect.c:3771
#: fe-connect.c:3908 fe-connect.c:3954 fe-connect.c:3972 fe-connect.c:4051
#: fe-connect.c:4121 fe-connect.c:4167 fe-connect.c:4185 fe-exec.c:3121
#: fe-exec.c:3286 fe-lobj.c:696 fe-protocol2.c:1027 fe-protocol3.c:1421
msgid "out of memory\n"
msgstr "メモリ不足です\n"

#: fe-auth.c:644
msgid "could not acquire SSPI credentials"
msgstr "SSPI資格を入手できませんでした"

#: fe-auth.c:736
msgid "SCM_CRED authentication method not supported\n"
msgstr "SCM_CRED認証方式はサポートされていません\n"

#: fe-auth.c:810
msgid "Kerberos 4 authentication not supported\n"
msgstr "Kerberos 4認証はサポートされていません\n"

#: fe-auth.c:826
msgid "Kerberos 5 authentication not supported\n"
msgstr "Kerberos 5認証はサポートされていません\n"

#: fe-auth.c:893
msgid "GSSAPI authentication not supported\n"
msgstr "GSSAPI認証はサポートされていません\n"

#: fe-auth.c:917
msgid "SSPI authentication not supported\n"
msgstr "SSPI認証はサポートされていません\n"

#: fe-auth.c:924
msgid "Crypt authentication not supported\n"
msgstr "Crypt認証はサポートされていません\n"

#: fe-auth.c:951
#, c-format
msgid "authentication method %u not supported\n"
msgstr "認証方式%uはサポートされていません\n"

#: fe-connect.c:712
#, c-format
msgid "invalid sslmode value: \"%s\"\n"
msgstr "sslmodeの値が無効です: \"%s\"\n"

#: fe-connect.c:733
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "SSLサポートが組み込まれていない場合sslmodeの値\"%s\"は無効です\n"

#: fe-connect.c:916
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "TCPソケットを非遅延モードに設定できませんでした: %s\n"

#: fe-connect.c:946
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running locally and accepting\n"
"\tconnections on Unix domain socket \"%s\"?\n"
msgstr ""
"サーバに接続できませんでした: %s\n"
"                              ローカルにサーバが稼動していますか?\n"
"                              Unixドメインソケット\"%s\"で通信を受け付けてい"
"ますか?\n"

#: fe-connect.c:956
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"サーバに接続できませんでした: %s\n"
"                              サーバはホスト\"%s\"で稼動していますか?\n"
"                              また、ポート%sでTCP/IP接続を受け付けています"
"か?\n"

#: fe-connect.c:1011
#, c-format
msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPIDLE)が失敗しました: %s\n"

#: fe-connect.c:1024
#, c-format
msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPALIVE)が失敗しました: %s\n"

#: fe-connect.c:1056
#, c-format
msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
msgstr "setsockopt(TCP_KEEPINTVL)が失敗しました: %s\n"

#: fe-connect.c:1088
#, c-format
msgid "setsockopt(TCP_KEEPCNT) failed: %s\n"
msgstr "setsockopt(TCP_KEEPCNT)が失敗しました: %s\n"

#: fe-connect.c:1137
#, c-format
msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
msgstr "WSAIoctl(SIO_KEEPALIVE_VALS)に失敗しました:%ui\n"

#: fe-connect.c:1189
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "無効なポート番号です: \"%s\"\n"

#: fe-connect.c:1231
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "ホスト名\"%s\"をアドレスに変換できませんでした: %s\n"

#: fe-connect.c:1235
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr "Unixドメインソケットのパス\"%s\"をアドレスに変換できませんでした: %s\n"

#: fe-connect.c:1444
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "接続状態が無効です。メモリ障害の可能性があります\n"

#: fe-connect.c:1487
#, c-format
msgid "could not create socket: %s\n"
msgstr "ソケットを作成できませんでした: %s\n"

#: fe-connect.c:1510
#, c-format
msgid "could not set socket to non-blocking mode: %s\n"
msgstr "ソケットを非ブロッキングモードに設定できませんでした: %s\n"

#: fe-connect.c:1522
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr "ソケットをclose-on-execモードに設定できませんでした: %s\n"

#: fe-connect.c:1540
msgid "keepalives parameter must be an integer\n"
msgstr "keepaliveのパラメータは整数でなければなりません\n"

#: fe-connect.c:1553
#, c-format
msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(SO_KEEPALIVE)が失敗しました: %s\n"

#: fe-connect.c:1694
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "ソケットのエラー状態を入手できませんでした: %s\n"

#: fe-connect.c:1732
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "ソケットからクライアントアドレスを入手できませんでした: %s\n"

#: fe-connect.c:1776
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "SSL調停パケットを送信できませんでした: %s\n"

#: fe-connect.c:1815
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "開始パケットを送信できませんでした: %s\n"

#: fe-connect.c:1882 fe-connect.c:1901
msgid "server does not support SSL, but SSL was required\n"
msgstr "サーバはSSLをサポートしていませんが、SSLが要求されました\n"

#: fe-connect.c:1917
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "SSL調停に対して無効な応答を受信しました: %c\n"

#: fe-connect.c:1993 fe-connect.c:2026
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "サーバからの認証要求を想定していましたが、%cを受信しました\n"

#: fe-connect.c:2197
#, c-format
msgid "out of memory allocating GSSAPI buffer (%i)"
msgstr "GSSAPIバッファの割り当て時のメモリ不足(%i)"

#: fe-connect.c:2282
msgid "unexpected message from server during startup\n"
msgstr "起動時にサーバから想定外のメッセージがありました\n"

#: fe-connect.c:2378
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "接続状態%dが無効です。メモリ障害の可能性があります\n"

#: fe-connect.c:2737 fe-connect.c:2797
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "PGEVT_CONNRESETイベント中にPGEventProc \"%s\"に失敗しました\n"

#: fe-connect.c:3142
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "無効なLDAP URL\"%s\":スキーマはldap://でなければなりません\n"

#: fe-connect.c:3157
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "無効なLDAP URL \"%s\": 区別名がありません\n"

#: fe-connect.c:3168 fe-connect.c:3221
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "無効なLDAP URL \"%s\": 正確に1つの属性を持たなければなりません\n"

#: fe-connect.c:3178 fe-connect.c:3235
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr ""
"無効なLDAP URL \"%s\": 検索スコープ(base/one/sub)を持たなければなりません\n"

#: fe-connect.c:3189
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "無効なLDAP URL \"%s\": フィルタがありません\n"

#: fe-connect.c:3210
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "無効なLDAP URL \"%s\": ポート番号が無効です\n"

#: fe-connect.c:3244
msgid "could not create LDAP structure\n"
msgstr "LDAP構造体を作成できませんでした: %s\n"

#: fe-connect.c:3286
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "LDAPサーバで検索に失敗しました: %s\n"

#: fe-connect.c:3297
msgid "more than one entry found on LDAP lookup\n"
msgstr "LDAP検索結果が複数ありました\n"

#: fe-connect.c:3298 fe-connect.c:3310
msgid "no entry found on LDAP lookup\n"
msgstr "LDAP検索結果が空でした\n"

#: fe-connect.c:3321 fe-connect.c:3334
msgid "attribute has no values on LDAP lookup\n"
msgstr "LDAP検索で属性に値がありませんでした\n"

#: fe-connect.c:3385 fe-connect.c:3403 fe-connect.c:3810
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "接続情報文字列において\"%s\"の後に\"=\"がありませんでした\n"

#: fe-connect.c:3466 fe-connect.c:3892 fe-connect.c:4076
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "接続オプション\"%s\"は無効です\n"

#: fe-connect.c:3479 fe-connect.c:3859
msgid "unterminated quoted string in connection info string\n"
msgstr "接続情報文字列において閉じていない引用符がありました\n"

#: fe-connect.c:3518
msgid "could not get home directory to locate service definition file"
msgstr ""
"サーバ設定ファイルの場所を特定しようとしましたが、ホームディレクトリを取得で"
"きませんでした。"

#: fe-connect.c:3551
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "サービス定義\"%s\"がみつかりません\n"

#: fe-connect.c:3574
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "サービスファイル\"%s\"がみつかりません\n"

#: fe-connect.c:3587
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "サービスファイル\"%2$s\"の行%1$dが長すぎます。\n"

#: fe-connect.c:3658 fe-connect.c:3685
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "サービスファイル\"%s\"の行%dに構文エラーがあります\n"

#: fe-connect.c:4352
msgid "connection pointer is NULL\n"
msgstr "接続ポインタはNULLです\n"

#: fe-connect.c:4625
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "WARNING: パスワードファイル\"%s\"がテキストファイルではありません\n"

#: fe-connect.c:4634
#, c-format
msgid ""
"WARNING: password file \"%s\" has group or world access; permissions should "
"be u=rw (0600) or less\n"
msgstr ""
"警告: パスワードファイル \"%s\" がグループメンバもしくは他のユーザから読める"
"状態になっています。この権限はu=rw (0600)以下にすべきです\n"

#: fe-connect.c:4722
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "ファイル\"%s\"からパスワードを読み込みました\n"

#: fe-exec.c:827
msgid "NOTICE"
msgstr "注意"

#: fe-exec.c:1014 fe-exec.c:1071 fe-exec.c:1111
msgid "command string is a null pointer\n"
msgstr "コマンド文字列がヌルポインタです\n"

#: fe-exec.c:1104 fe-exec.c:1199
msgid "statement name is a null pointer\n"
msgstr "文の名前がヌルポインタです\n"

#: fe-exec.c:1119 fe-exec.c:1273 fe-exec.c:1928 fe-exec.c:2125
msgid "function requires at least protocol version 3.0\n"
msgstr "関数は少なくともプロトコルバージョン3.0が必要です\n"

#: fe-exec.c:1230
msgid "no connection to the server\n"
msgstr "サーバへの接続がありません\n"

#: fe-exec.c:1237
msgid "another command is already in progress\n"
msgstr "他のコマンドを処理しています\n"

#: fe-exec.c:1349
msgid "length must be given for binary parameter\n"
msgstr "バイナリパラメータには長さを指定しなければなりません\n"

#: fe-exec.c:1596
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "想定外のasyncStatus: %d\n"

#: fe-exec.c:1616
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "PGEVT_RESULTCREATEイベント中にPGEventProc \"%s\"に失敗しました\n"

#: fe-exec.c:1746
msgid "COPY terminated by new PQexec"
msgstr "新たなPQexec\"によりCOPYが終了しました"

#: fe-exec.c:1754
msgid "COPY IN state must be terminated first\n"
msgstr "まずCOPY IN状態を終了させなければなりません\n"

#: fe-exec.c:1774
msgid "COPY OUT state must be terminated first\n"
msgstr "まずCOPY OUT状態を終了させなければなりません\n"

#: fe-exec.c:2016 fe-exec.c:2082 fe-exec.c:2167 fe-protocol2.c:1172
#: fe-protocol3.c:1557
msgid "no COPY in progress\n"
msgstr "実行中のCOPYはありません\n"

#: fe-exec.c:2359
msgid "connection in wrong state\n"
msgstr "接続状態が異常です\n"

#: fe-exec.c:2390
msgid "invalid ExecStatusType code"
msgstr "ExecStatusTypeコードが無効です"

#: fe-exec.c:2454 fe-exec.c:2477
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "列番号%dは0..%dの範囲を超えています"

#: fe-exec.c:2470
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "行番号%dは0..%dの範囲を超えています"

#: fe-exec.c:2492
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "パラメータ%dは0..%dの範囲を超えています"

#: fe-exec.c:2780
#, c-format
msgid "could not interpret result from server: %s"
msgstr "サーバからの結果を解釈できませんでした: %s"

#: fe-exec.c:3019 fe-exec.c:3103
msgid "incomplete multibyte character\n"
msgstr "不完全なマルチバイト文字\n"

#: fe-lobj.c:152
msgid "cannot determine OID of function lo_truncate\n"
msgstr "lo_truncate関数のOIDを決定できません\n"

#: fe-lobj.c:380
msgid "cannot determine OID of function lo_create\n"
msgstr "lo_create関数のOIDを決定できません\n"

#: fe-lobj.c:525 fe-lobj.c:624
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "ファイル\"%s\"をオープンできませんでした: %s\n"

#: fe-lobj.c:575
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "ファイル\"%s\"を読み込めませんでした: %s\n"

#: fe-lobj.c:639 fe-lobj.c:663
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "ファイル\"%s\"に書き込めませんでした: %s\n"

#: fe-lobj.c:744
msgid "query to initialize large object functions did not return data\n"
msgstr ""
"ラージオブジェクト機能を初期化する問い合わせがデータを返しませんでした\n"

#: fe-lobj.c:785
msgid "cannot determine OID of function lo_open\n"
msgstr "lo_open関数のOIDを決定できません\n"

#: fe-lobj.c:792
msgid "cannot determine OID of function lo_close\n"
msgstr "lo_close関数のOIDを決定できません\n"

#: fe-lobj.c:799
msgid "cannot determine OID of function lo_creat\n"
msgstr "lo_creat関数のOIDを決定できません\n"

#: fe-lobj.c:806
msgid "cannot determine OID of function lo_unlink\n"
msgstr "lo_unlink関数のOIDを決定できません\n"

#: fe-lobj.c:813
msgid "cannot determine OID of function lo_lseek\n"
msgstr "lo_lseek関数のOIDを決定できません\n"

#: fe-lobj.c:820
msgid "cannot determine OID of function lo_tell\n"
msgstr "lo_tell関数のOIDを決定できません\n"

#: fe-lobj.c:827
msgid "cannot determine OID of function loread\n"
msgstr "loread関数のOIDを決定できません\n"

#: fe-lobj.c:834
msgid "cannot determine OID of function lowrite\n"
msgstr "lowrite関数のOIDを決定できません\n"

#: fe-misc.c:262
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "サイズ%luの整数はpqGetIntでサポートされていません"

#: fe-misc.c:298
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "サイズ%luの整数はpqPutIntでサポートされていません"

#: fe-misc.c:578 fe-misc.c:780
msgid "connection not open\n"
msgstr "接続はオープンされていません\n"

#: fe-misc.c:643 fe-misc.c:733
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "サーバからデータを受信できませんでした: %s\n"

#: fe-misc.c:750 fe-misc.c:828
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"サーバとの接続が想定外にクローズされました\n"
"        おそらく要求の処理前または処理中にサーバが異常終了\n"
"        したことを意味しています。\n"

#: fe-misc.c:845
#, c-format
msgid "could not send data to server: %s\n"
msgstr "サーバにデータを送信できませんでした: %s\n"

#: fe-misc.c:964
msgid "timeout expired\n"
msgstr "タイムアウト期間が過ぎました\n"

#: fe-misc.c:1009
msgid "socket not open\n"
msgstr "ソケットがオープンされていません\n"

#: fe-misc.c:1032
#, c-format
msgid "select() failed: %s\n"
msgstr "select()が失敗しました: %s\n"

#: fe-protocol2.c:89
#, c-format
msgid "invalid setenv state %c, probably indicative of memory corruption\n"
msgstr "setenv状態%cは無効です。メモリ障害の可能性があります\n"

#: fe-protocol2.c:330
#, c-format
msgid "invalid state %c, probably indicative of memory corruption\n"
msgstr "状態%cは無効です。メモリ障害の可能性があります\n"

#: fe-protocol2.c:419 fe-protocol3.c:186
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "待機中にサーバからメッセージ種類0x%02xが届きました"

#: fe-protocol2.c:462
#, c-format
msgid "unexpected character %c following empty query response (\"I\" message)"
msgstr "空の問い合わせ応答(\"I\"メッセージ)の後に想定外の文字%cがありました"

#: fe-protocol2.c:516
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"サーバが事前の行記述(\"T\"メッセージ)なしにデータ(\"D\"メッセージ)を送信しま"
"した"

#: fe-protocol2.c:532
msgid ""
"server sent binary data (\"B\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"サーバが事前の行記述(\"T\"メッセージ)なしにバイナリデータ(\"B\"メッセージ)を"
"送信しました"

#: fe-protocol2.c:547 fe-protocol3.c:382
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "サーバから想定外の応答がありました。受け付けた先頭文字は\"%c\"です\n"

#: fe-protocol2.c:768 fe-protocol3.c:701
msgid "out of memory for query result\n"
msgstr "問い合わせ結果用のメモリが不足しています\n"

#: fe-protocol2.c:1215 fe-protocol3.c:1625
#, c-format
msgid "%s"
msgstr "%s"

#: fe-protocol2.c:1227
msgid "lost synchronization with server, resetting connection"
msgstr "サーバとの動機が失われました。接続をリセットしています"

#: fe-protocol2.c:1361 fe-protocol2.c:1393 fe-protocol3.c:1828
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "プロトコルエラー: id=0x%x\n"

#: fe-protocol3.c:344
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)\n"
msgstr ""
"サーバが事前の行記述(\"T\"メッセージ)なしにデータ(\"D\"メッセージ)を送信しま"
"した\"\n"

#: fe-protocol3.c:403
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "メッセージの内容がメッセージ種類\"%c\"の長さに合いません\n"

#: fe-protocol3.c:424
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr ""
"サーバとの同期が失われました。受信したメッセージ種類は\"%c\"、長さは%d\n"

#: fe-protocol3.c:646
msgid "unexpected field count in \"D\" message\n"
msgstr "\"D\"メッセージ内のフィールド数が想定外でした\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:788 fe-protocol3.c:807
#, c-format
msgid " at character %s"
msgstr "(文字位置: %s)"

#: fe-protocol3.c:820
#, c-format
msgid "DETAIL:  %s\n"
msgstr "DETAIL:  %s\n"

#: fe-protocol3.c:823
#, c-format
msgid "HINT:  %s\n"
msgstr "HINT:  %s\n"

#: fe-protocol3.c:826
#, c-format
msgid "QUERY:  %s\n"
msgstr "QUERY:  %s\n"

#: fe-protocol3.c:829
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "CONTEXT:  %s\n"

#: fe-protocol3.c:841
msgid "LOCATION:  "
msgstr "LOCATION:  "

#: fe-protocol3.c:843
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:845
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1069
#, c-format
msgid "LINE %d: "
msgstr "行 %d: "

#: fe-protocol3.c:1453
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline: テキストのCOPY OUTを行っていません\n"

#: fe-secure.c:265
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "SSL接続を確立できませんでした: %s\n"

#: fe-secure.c:349 fe-secure.c:436 fe-secure.c:1162
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "SSL SYSCALLエラー: %s\n"

#: fe-secure.c:355 fe-secure.c:442 fe-secure.c:1166
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "SSL SYSCALLエラー: EOFを検知\n"

#: fe-secure.c:367 fe-secure.c:453 fe-secure.c:1175
#, c-format
msgid "SSL error: %s\n"
msgstr "SSLエラー: %s\n"

#: fe-secure.c:377 fe-secure.c:463 fe-secure.c:1184
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "不明のSSLエラーコード: %d\n"

#: fe-secure.c:601
msgid ""
"verified SSL connections are only supported when connecting to a host name\n"
msgstr "検証されたSSL接続はホスト名への接続時のみサポートされます\n"

#: fe-secure.c:620
#, c-format
msgid "server common name \"%s\" does not match host name \"%s\"\n"
msgstr "サーバの正式名(common name)\"%s\"がホスト名\"%s\"と一致しません\n"

#: fe-secure.c:752
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "SSLコンテキストを作成できませんでした: %s\n"

#: fe-secure.c:843
msgid "could not get home directory to locate client certificate files\n"
msgstr ""
"クライアント証明書ファイルの場所を特定しようとしましたが、ホームディレクトリ"
"を取得できませんでした\n"

#: fe-secure.c:868
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "証明書ファイル\"%s\"をオープンできませんでした: %s\n"

#: fe-secure.c:893 fe-secure.c:903
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "証明書ファイル\"%s\"を読み込めませんでした: %s\n"

#: fe-secure.c:940
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "SSLエンジン\"%s\"を読み込みできませんでした: %s\n"

#: fe-secure.c:952
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "SSLエンジン\"%s\"を初期化できませんでした: %s\n"

#: fe-secure.c:968
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr ""
"SSL秘密キーファイル\"%s\"をエンジン\"%s\"から読み取れませんでした: %s\n"

#: fe-secure.c:982
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "SSL秘密キー\"%s\"をエンジン\"%s\"から読み取れませんでした: %s\n"

#: fe-secure.c:1017
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "証明書はありましたが、秘密キーファイル\"%s\"はありませんでした\n"

#: fe-secure.c:1025
#, c-format
msgid ""
"private key file \"%s\" has group or world access; permissions should be "
"u=rw (0600) or less\n"
msgstr ""
"警告：秘密キーファイル \"%s\" がグループメンバや第三者から読める状態になって"
"います。この権限はu=rw (0600)またはそれ以下とすべきです\n"

#: fe-secure.c:1036
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "秘密キーファイル\"%s\"をロードできませんでした: %s\n"

#: fe-secure.c:1050
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "証明書と秘密キーファイル\"%s\"が一致しません: %s\n"

#: fe-secure.c:1075
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "ルート証明書\"%s\"を読み取れませんでした: %s\n"

#: fe-secure.c:1099
#, c-format
msgid "SSL library does not support CRL certificates (file \"%s\")\n"
msgstr "SSLライブラリがCRL証明書（\"%s\"）をオープンできませんでした\n"

#: fe-secure.c:1120
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"ルート証明書ファイル\"%s\"が存在しません。\n"
"ファイルを用意するかサーバ証明書の検証を無効にするようにsslmodeを変更してくだ"
"さい\n"

#: fe-secure.c:1203
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "証明書を入手できませんでした: %s\n"

#: fe-secure.c:1231
msgid "SSL certificate's common name contains embedded null\n"
msgstr "SSL 証明書のコモンネームに null が含まれています\n"

#: fe-secure.c:1307
msgid "no SSL error reported"
msgstr "SSLエラーはありませんでした"

#: fe-secure.c:1316
#, c-format
msgid "SSL error code %lu"
msgstr "SSLエラーコード: %lu"
